### PR TITLE
small change to allow adding of a team to a repo

### PR DESCRIPTION
**Pull request by michaelneale**
@charleso @markhibberd review pls - needed for https://github.com/ambiata/anatomy/pull/26

### CODE REVIEW

File: Github/Organizations.hs [hunk 1]
```diff
@@ -33,3 +34,7 @@ publicOrganization' auth reqOrganizationName = githubGet' auth ["orgs", reqOrgan
 -- > publicOrganization "thoughtbot"
 publicOrganization :: String -> IO (Either Error Organization)
 publicOrganization = publicOrganization' Nothing
+
+-- PUT /teams/:id/repos/:org/:repo
+addTeamToRepo auth teamId orgName projectName = do
```
Comment (charleso) [inline Github/Organizations.hs]: Function signature types /nit

**Comment by charleso** [comment]
:+1:

**Comment by markhibberd** [comment]
:-1:

**Comment by markhibberd** [comment]
:+1:

**Comment by markhibberd** [comment]
Unfortunate typo.

**Pull request merged**